### DISCUSSION TITLE
Help Installing

### DISCUSSION THREAD

**Discussion by fkan2500** [cmu-sei/crucible#71] [General]
Good evening, I want to use this for my classroom but the instructions are very confusing and it feels like installing this requires a very high level of working knowledge that I just don't have. the page has multiple apps listed but there is no listing on how to install crucible frame work. I also tried foundry but when I ran the esxi set-up I would get an error. any help would be greatly appreciated. Thanks.

**Comment by JuliaJin05** [comment]
Hi, I was also struggling a little since I wasn't sure if we had to manually set up each UI and API which there were about 15. Is there any completed OVA file available anywhere? Thank you

**Comment by sei-rlehman** [comment]
@fkan2500 Thank you for your interest in Crucible!  We realize that the framework installation asks a lot of a would-be installer right now - we're working on that!  We are currently developing better documentation for our [helm charts](https://github.com/cmu-sei/helm-charts) to assist with setting up Crucible with Kubernetes. I will try to update you here when we make that available on our [documentation site](https://cmu-sei.github.io/crucible/).

@JuliaJin05 We have a Foundry Appliance repo with a [Crucible branch](https://github.com/cmu-sei/foundry-appliance/tree/crucible) that you can use to build a single-VM instance of Crucible. Give it a try and let us know if you need help. We plan on spinning this work off into its own repo sometime in the near future.
